download.md

```diff
@@ -3,39 +3,26 @@ title: Downloads
 ---
 
 ## Current Version
-The current version of Jython is 2.7.3.
-It can be downloaded here:
-- [Jython Installer](https://repo1.maven.org/maven2/org/python/jython-installer/2.7.3/jython-installer-2.7.3.jar):
-  Use this to install Jython.
-  ([metadata](https://search.maven.org/artifact/org.python/jython-installer/2.7.3/jar))
-- [Jython Standalone](https://repo1.maven.org/maven2/org/python/jython-standalone/2.7.3/jython-standalone-2.7.3.jar):
-  Use this to run Jython without installing or to embed Jython in a Java application.
-  ([metadata](https://search.maven.org/artifact/org.python/jython-standalone/2.7.3/jar))
-- You may cite Jython 2.7.3 as a
-  [dependency in your Maven or Gradle build](https://search.maven.org/artifact/org.python/jython-slim/2.7.3/jar).
+The current version of Jython is 2.7.4.
+The current version can be used:
```

Comment:
looks/sounds repetitive "The current version" twice. I preferred the old formulation.

Reply:
Perhaps, just "It can be used:"